packages/eslint-plugin/.eslint-doc-generatorrc.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import('eslint-doc-generator').GenerateOptions} */
+const config = {
+  configEmoji: [
+    ['esm', '📁'],
+    ['typescript', '🧠'],
+    ['react', '🏡'],
```

Comment:
open to better emoij suggestions for these

Reply:
```suggestion
    ['esm', '📁'],
    ['typescript', '🧠'],
    ['react', '⚛️'],
```